staging/src/k8s.io/apiserver/pkg/storage/testing/store_tests.go

```diff
@@ -769,3 +769,13 @@ func ExpectNoDiff(t *testing.T, msg string, expected, got interface{}) {
 		}
 	}
 }
+
+const dummyPrefix = "adapter"
```

Comment:
minor nit - can we move this to some "util.go" file or sth like that in this directory?

It would be cleaner to leave this file for tests themselves...